rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -365,53 +372,64 @@ public String getMBeanInfo(String sessionId, String name, List<String> attrs)
+                                                                                                               objectName,
+                                                                                                               attrs,
+                                                                                                               timeRange));
     }
 
     @Override
     public String getNodeMBeansInfo(String sessionId, String nodeJmxUrl, String objectNames, List<String> attrs)
             throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.getNodeMBeansInfo(sessionId,
-                                                                                              nodeJmxUrl,
-                                                                                              objectNames,
-                                                                                              attrs));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.getNodeMBeansInfo(sessionId,
+                                                                                                             nodeJmxUrl,
+                                                                                                             objectNames,
+                                                                                                             attrs));
     }
 
     @Override
     public String getNodeMBeansHistory(String sessionId, String nodeJmxUrl, String objectNames, List<String> attrs,
             String timeRange) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.getNodeMBeansHistory(sessionId,
-                                                                                                 nodeJmxUrl,
-                                                                                                 objectNames,
-                                                                                                 attrs,
-                                                                                                 timeRange));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.getNodeMBeansHistory(sessionId,
+                                                                                                                nodeJmxUrl,
+                                                                                                                objectNames,
+                                                                                                                attrs,
+                                                                                                                timeRange));
     }
 
     @Override
     public String getStatHistory(String sessionId, String range) throws RestServerException, ServiceException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)